manage.py

```diff
@@ -257,18 +257,18 @@ def import_commands():
     """Imports commands from the CommandDocs.txt file in ex2_ground_station_software.
     """
     try:
-        with open("ex2_ground_station_software/CommandDocs.txt", 'r') as f:
+        with open('ex2_ground_station_software/CommandDocs.txt', 'r') as f:
             text = f.read()
     except FileNotFoundError:
-        print("Couldn't find list of commands at ex2_ground_station_software/CommandDocs.txt. Seeding database with no commands.")
+        print('Couldn't find list of commands at ex2_ground_station_software/CommandDocs.txt. Seeding database with no commands.')
```

Comment:
```suggestion
        print('Couldn\'t find list of commands at ex2_ground_station_software/CommandDocs.txt. Seeding database with no commands.')
```
Bug fix for escaping the `'` character